### PR TITLE
[tyr] Loki: enrich ntfs with addresses/admins

### PR DESCRIPTION
**Pull request by datanel**
https://navitia.atlassian.net/browse/NAV-2217?page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-53940

Just before putting the NTFS to S3 for Loki, it is enriched with addresses and administrative regions calling bragi.
For that an intermediate folder `ntfs_with_addreses` is created

### CODE REVIEW

**Review by azime — APPROVED** [review]

File: source/tyr/tyr/binarisation.py [hunk 1]
```diff
@@ -1125,9 +1125,47 @@ def poi2mimir(self, instance_name, input, autocomplete_version, job_id=None, dat
...
+
+    logger = get_instance_logger(instance, task_id=job_id)
+    filename = zip_if_needed(filename)
+
+    try:
+        params = [
+            "--input",
+            filename,
+            "--output",
+            filename,
```
Comment (pbench) [inline source/tyr/tyr/binarisation.py]: I am not sure if this is a good idea to replace the input ntfs with the enriched one.
Maybe write to a different location ?

Comment (datanel) [reply source/tyr/tyr/binarisation.py]: it works and it should not be a problem imho.
if there is a problem, filename should not be atered by the output.
if the job succeed, the ntfs is just replaced.

Comment (datanel) [reply source/tyr/tyr/binarisation.py]: ok I see, the ntfs is sent to fusio2ed after, we should keep the original; ok

Comment (pbench) [reply source/tyr/tyr/binarisation.py]: Yeah I don't know exactly what happen with the ntfs (nor when), but I know that *some* things can happen (sent to fusio2ed, moved to a backup directory, ... ?). 
So better be safe than sorry :)

Comment (datanel) [reply source/tyr/tyr/binarisation.py]: fixed in the last commit.
How can I increase the code coverage ? Sonar is warning

File: source/tyr/tyr/binarisation.py [hunk 2]
```diff
@@ -1125,9 +1125,53 @@ def poi2mimir(self, instance_name, input, autocomplete_version, job_id=None, dat
...
+            "--bragi-url",
+            current_app.config['BRAGI_URL'],
+        ]
+
+        res = None
+        with collect_metric("enrich-ntfs-with-addresses", job, dataset_uid):
+            res = launch_exec("enrich-ntfs-with-addresses", params, logger)
+        if res != 0:
+            raise ValueError("enrich-ntfs-with-addresses failed")
+        dataset.state = "done"
```
Comment (pbench) [inline source/tyr/tyr/binarisation.py]: Not sure when we should set the `dataset.state` to `done` ?

**Review by pbench — APPROVED** [review]

**Pull request merged**